martin-ejercicio2.c

```diff
@@ -2,12 +2,17 @@
 
 int main(int argc, char *argv[]) {
 	
-	// el mismo codigo con un comentario 	
-	for(int i=-9; i < 10; i++){
```

Comment:
me faltó bajar las llaves del main y del for para que sea consistente con el resto del codigo

Reply:
genial